File: YaroWilliams/Exercise25.java
import java.util.Scanner;
public class Exercise25 {
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);

        int coverage = 5;

        System.out.println("Enter the height of your wall");
        int height = input.nextInt();

        System.out.println("Enter the width of your wall");
        int width = input.nextInt();

        int numOfCans = (int) Math.ceil(((height * width) / coverage)) ;

        System.out.println("You need "+numOfCans + " cans of paint to paint the wall");
        
    }
}

File: Stephannie/Exercise-25.py
import math
#paint area calculator

###A can can paint 5m*m area

Height = float(input("What is the height of the wall?: "))
Width = float(input("What is the width of the wall?: "))

area = int(Height * Width)
print(f"Your total area is {area}")

number_of_paints = math.ceil(area / 5) 

print(f"You will need {number_of_paints} cans of paint to paint the wall.")

